README.md

```diff
@@ -36,7 +36,9 @@ TLDR: OpenShift can generate and expose environment variables to our application
 
 ###Development mode
 When you develop your Rails application in OpenShift, you can also enable the 'development' environment by setting the RAILS_ENV environment variable for your deploymentConfiguration, using the `oc` client, like:  
-`oc env dc/frontend RAILS_ENV=development`  
+
+		$ oc env dc/frontend RAILS_ENV=development
```

Comment:
I don't see how this could be useful for users that run Rails apps in OpenShift. If you set this to development then most likely you won't be able to connect to database (or you connect to 'default' which is production). The other thing what development mode sets up is dynamic Ruby class reloading, which means that you can edit the Ruby source code and have it reloaded... However, we discourage users doing that in containers...

@bparees I would rather nuke the 'development mode' section from here to avoid confusion.